### PR TITLE
Add Encounter External Indicator Documentation

### PR DESCRIPTION
**Pull request by sydneanschutz**
Terminologies:
Screen Shot 2019-12-17 at 2 41 07 PM
Screen Shot 2019-12-17 at 2 40 56 PM

Create: 
Screen Shot 2019-12-17 at 2 40 36 PM

### CODE REVIEW

File: lib/resources/r4/encounter.yaml [hunk 1]
```diff
@@ -169,6 +169,13 @@ fields:
     - display: Millennium Encounter Visit Type
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/71
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-71-visit-type
+    - display: v2 Patient Class
```
Comment (ebaranowski) [inline lib/resources/r4/encounter.yaml]: I dont understand where this is coming from....

http://hl7.org/fhir/valueset-encounter-type.html

File: lib/resources/r4/encounter.yaml [hunk 2]
```diff
@@ -544,3 +551,32 @@ fields:
       {
         "reference": "Location/35646787"
       }
+
+- name: meta.security
+  url: http://hl7.org/fhir/resource-definitions.html#Meta.security
```
Comment (ebaranowski) [inline lib/resources/r4/encounter.yaml]: R4

File: lib/resources/r4/encounter.yaml (hunk 1 above)
Comment (ebaranowski) [reply lib/resources/r4/encounter.yaml]: Talked offline with Kathy. The value set is an example value set

File: lib/resources/r4/encounter.yaml (hunk 2 above)
Comment (sydneanschutz) [reply lib/resources/r4/encounter.yaml]: https://github.com/cerner/fhir.cerner.com/pull/333/commits/ecb183ebe47aae56128b75b511fccd7a46f190fc

**Comment by ebaranowski** [comment]
Screen Shot 2019-12-17 at 3 03 55 PM
Screen Shot 2019-12-17 at 3 04 08 PM
Validated

**Comment by averyallen** [comment]
image

**Pull request merged**